templates/cws/cws_deployment.tmpl

```diff
@@ -0,0 +1,228 @@
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: portal-x
+      app.kubernetes.io/name: cws-test
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: portal-x
+        app.kubernetes.io/name: cws-test
+        app: cws-test
+    spec:
+      containers:
+      - args:
+        - server
+        - --debug
+        - --database
+        - $(DATABASE)
+        envFrom:
+        - secretRef:
+            name: customer-web-server-secret
+        image: mattermost/cws-test:{{ .ImageTag }}
+        imagePullPolicy: Always
+        name: cws-main
+        ports:
+        - containerPort: 8076
+          name: api
+          protocol: TCP
+        - containerPort: 8077
+          name: internal
+          protocol: TCP
+        resources: {}
+      imagePullSecrets:
+      - name: customer-web-server-gitlab-image
+      initContainers:
+      - args:
+        - schema
+        - migrate
+        - --database
+        - $(DATABASE)
+        envFrom:
+        - secretRef:
+            name: customer-web-server-secret
+        image: mattermost/cws-test:{{ .ImageTag }}
+        imagePullPolicy: Always
+        name: customer-web-server-init-database
+      restartPolicy: Always
+      schedulerName: default-scheduler
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: customer-web-server-secret
+  namespace: {{ .Namespace }}
+type: Opaque
+stringData:
+  DATABASE: {{ .Environment.Database }}
+  CWS_PAYMENT_URL: {{ .Environment.CWSPaymentURL }}
+  CWS_PAYMENT_TOKEN: {{ .Environment.CWSPaymentToken }}
+  CWS_SITEURL: {{ .Environment.CWSSiteURL }}
+  CWS_SMTP_USERNAME: {{ .Environment.CWSSMTPUsername }}
+  CWS_SMTP_PASSWORD: {{ .Environment.CWSSMTPPassword }}
+  CWS_SMTP_SERVER: {{ .Environment.CWSSMTPServer }}
+  CWS_SMTP_PORT: "{{ .Environment.CWSSMTPPort }}"
+  CWS_SMTP_SERVERTIMEOUT: "{{ .Environment.CWSSMTPServerTimeout }}"
+  CWS_SMTP_CONNECTIONSECURITY: {{ .Environment.CWSSMTPConnectionSecurity }}
+  CWS_EMAIL_REPLYTONAME: {{ .Environment.CWSEmailReplyToName }}
+  CWS_EMAIL_REPLYTOADDRESS: {{ .Environment.CWSEmailReplyToAddress }}
+  CWS_EMAIL_BCCADDRESSES: {{ .Environment.CWSEmailBCCAddress }}
+  CWS_CLOUD_URL: {{ .Environment.CWSCloudURL }}
+---
+apiVersion: v1
+data:
+  .dockerconfigjson: {{ .Environment.DockerHubCredentials }}
+kind: Secret
+metadata:
+  name: customer-web-server-gitlab-image
+  namespace: {{ .Namespace }}
+type: kubernetes.io/dockerconfigjson
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-config
+  namespace: {{ .Namespace }}
+  labels:
+    app: postgres
+data:
+  POSTGRES_DB: xcws
+  POSTGRES_USER: xcws
+  POSTGRES_PASSWORD: xcws!934XCWS
+---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: postgres-pv-volume
+  labels:
+    type: local
+    app: postgres
+spec:
+  storageClassName: gp2
+  capacity:
+    storage: 5Gi
```

Comment:
if we use the local disk we might need to reduce this size. otherwise, we can run out of disk and things cannot be scheduled

Reply:
or request a dinamic ebs volume

Reply:
Bumped down to 1Gi. I think that'll be ok?